scripts/import/import_cosmic.pl

```diff
@@ -217,7 +228,7 @@ sub get_equivalent_class {
   my $start = shift;
   my $end   = shift;
 
-  my @type_parts = split(' ',$type);
+  my @type_parts = split(',|\s',$type);
   $type = $type_parts[0];
```

Comment:
Some COSMIC variants have more than one class splited by ',' in this case only the first class is used. Is this the expected behavior? One example: `"Complex - compound substitution,Substitution - intronic"` after import class = sequence_alteration

Reply:
Yes, this is expected.